osia/cli.py

```diff
@@ -20,6 +20,7 @@
 from typing import List
 import coloredlogs
 
+from .config.config import AMD_ARCH, ARM_ARCH, x86_64_ARCH, AARCH64_ARCH, S390X_ARCH, PPC_ARCH
```

Comment:
To be honest I would reorder the name - ARCH_... so all the constants can be quickly found.